packages/flutter/test/material/slider_test.dart

```diff
@@ -870,15 +1036,16 @@ void main() {
       expect(
         sliderBox,
         paints
-          ..circle(x: 396.6802978515625, y: 16.0, radius: 8.0)
+          ..circle(x: 185.5457763671875, y: 16.0, radius: 8.0)
```

Comment:
These x values are a bit terrifying however I suppose they serve to confirm then if the nothing has really changed vis the slider's implementation then nothing has really changed.

Reply:
Yeah, I agree, but there's no "close to" comparison for those.  I almost implemented one...

Reply:
you can update the paints matcher to take matchers, iirc amir did that for some of them already.